frontend/src/app/components/wp-buttons/wp-status-button/wp-status-button.component.ts

```diff
@@ -35,6 +35,8 @@ import {Highlighting} from "core-components/wp-fast-table/builders/highlighting/
 import {HalResource} from "core-app/modules/hal/resources/hal-resource";
 import {WorkPackageCacheService} from "core-components/work-packages/work-package-cache.service";
 import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
+import {concatAll} from "rxjs/operators";
+import {combineLatest} from "rxjs";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/18004c90a5f714efbb5cb6c38a447bce2c7a1ec4/frontend/src/app/components/wp-buttons/wp-status-button/wp-status-button.component.ts#L40"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>